test/Microsoft.Web.LibraryManager.Vsix.Test/UI/Models/LibraryIdViewModelTests.cs

```diff
@@ -41,7 +51,7 @@ public async Task FilterCompletions_SearchTextDoesNotContainAtSign_ReturnAllMatc
         {
             var testObj = new LibraryIdViewModel(GetTestSearchService(), "test");
 
-            CompletionSet result = await testObj.GetCompletionSetAsync(caretIndex: 0);
```

Comment:
FYI, changing the caretIndex for the existing tests here is necessary.

GetCompletionSetAsync calls LibraryCatalog.GetLibraryCompletionSetAsync, which returns libraries which match the substring left of the caretPosition. For the tests which I've changed the caretIndex for, caretIndex = 0, which means that the substring is an empty string and therefore will return all library results (which coincidentally worked since all the added libraries were "test")

So I've changed the caretIndex in order to properly enable the filtering on these tests.

Reply:
Thanks for explaining